manifests/base/argo-server/argo-server-deployment.yaml

```diff
@@ -19,6 +19,13 @@ spec:
           ports:
             - name: web
               containerPort: 2746
+          resources:
+            limits:
```

Comment:
is it sufficient for the argo server? Memory looks very low. can we increase on the Limit side?

Reply:
I've not tested a container yet - want to check with @jessesuen if we should do this at all.